engine/runtime/src/bench/java/org/enso/interpreter/bench/BenchmarksRunner.java

```diff
@@ -39,7 +41,8 @@ public BenchmarkItem run(String label) throws RunnerException, JAXBException {
     if (Boolean.getBoolean("bench.compileOnly")) {
       builder
         .measurementIterations(1)
-        .warmupIterations(0);
+        .warmupIterations(0)
+        .forks(0);
```

Comment:
Setting forks to 0 when `bench.CompileOnly = true` seems to have a negative side effect when all the benchmarks are run on the CI - it seems to cause StackOverflow errors, like [this one](https://github.com/enso-org/enso/actions/runs/5762870936/job/15623476613#step:10:1624)

Reply:
I have reverted 11e0a12c6fc449afede4a325659e071daef0104c as a workaround for now. In the upcoming PRs, I will try to unify and improve the debugging experience of the benchmarks. Tracking this in #7489.